tools/lint/item.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
```

Comment:
This tool is integrated from W3C, the license should not be Intel, please create a COPYING file to record the license.